src/Ben.Demystifier/EnhancedStackTrace.cs

```diff
@@ -101,12 +101,13 @@ internal void Append(StringBuilder sb)
 
                 sb.Append("   at ");
                 frame.MethodInfo.Append(sb);
-
-                var filePath = frame.GetFileName();
-                if (!string.IsNullOrEmpty(filePath))
+                
+                if (frame.GetFileName() is {} fileName
```

Comment:
This pattern same as `is not null`?

Reply:
Yes but here I wanted to have the value on variable so the null checker would stop whining.
I'm not sure why the `IsNullOrEmpty` wasn't enough for it to know it was safe to access `fileName`.

Reply:
Yes, theoretically you could get rid of the `IsNullOrEmpty` by using 
```csharp
if (frame.GetFileName() is { Length: > 0 } fileName)
```